tools/cli.py

```diff
@@ -25,31 +25,36 @@ def get_argument_list(self):
         frames_and_faces_dir = (" Must Pass in a frames folder/source video file AND a faces "
                                 "folder (-fr and -fc).")
         output_opts = " Use the output option (-o) to process results."
-        align_eyes = " Can optionally use the align-eyes switch (-ae)."
         argument_list = list()
         argument_list.append({
             "opts": ("-j", "--job"),
             "action": Radio,
             "type": str,
-            "choices": ("draw", "extract", "extract-large", "manual", "merge",
-                        "missing-alignments", "missing-frames", "legacy", "leftover-faces",
-                        "multi-faces", "no-faces", "reformat", "remove-faces", "remove-frames",
-                        "rename", "sort-x", "sort-y", "spatial", "update-hashes"),
+            "choices": ("draw", "extract", "extract-large", "extract-aligneyes", "manual",
```

Comment:
Could you move this back to an extract option please?

2 reasons:
1) The number of job options is already inordinately large, so would rather avoid adding more options here (I may change extract-large to be an extract option too)
2) We just convert it to a bool anyway, which is what the option will come in as, so there is no significant additional coding to be honest.

Reply:
I'm confused? Making it an extraction option defeats the whole purpose of deprecating it from the extract options and extract toolchain ?

User feedback was to add it to tools if were removing it from extract options, so I added it here.

Reply:
I mean in the align tools menu (it currently resides in there as an option)

Reply:
![image](https://user-images.githubusercontent.com/36920800/65068479-a6dc2f80-d980-11e9-9081-c49335d09a5a.png)

Reply:
yup, realized.. will do